cmd/otelcol/main.go

```diff
@@ -22,6 +22,7 @@ import (
 	"log"
 	"os"
 
+	_ "github.com/joho/godotenv/autoload"
```

Comment:
What is the behavior of invalid .env content and would it be better to use the library directly to marshal the error to a proper handler, like in main or `internal/settings`?

Reply:
imo I'd expect there to be a logged warning about the validity of the file w/ the parsing error and for none of the env vars in the file to be sourced but can see other options being desirable. Terminating the collector seems overkill.

Reply:
thanks, I'll factor that in.